src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -75,114 +75,24 @@
      * message in the "messages.properties" file.
      */
     public static final String CREATE_LISTENER_EXCEPTION = "Main.createListener";
+
+    /** Name for the moduleConfig attribute 'tabWidth'. */
+    private static final String ATTRIB_TAB_WIDTH_NAME = "tabWidth";
+
     /** Logger for Main. */
     private static final Log LOG = LogFactory.getLog(Main.class);
 
     /** Width of CLI help option. */
     private static final int HELP_WIDTH = 100;
```

Comment:
please move this constant to CliOptions class, to let this class own all about cli.

Reply:
Done: Moved HELP_WIDTH to CliOptions also.